fedbiomed/common/config.py

```diff
@@ -160,43 +147,36 @@ def generate(
         if self.is_config_existing() and not force:
             return self.read()
 
-
         # Create default section
         component_id = id if id else f"{self._COMPONENT_TYPE}_{uuid.uuid4()}"
 
-        self._cfg['default'] = {
-            'id': component_id,
-            'component': self._COMPONENT_TYPE,
-            'version': str(self._CONFIG_VERSION)
+        self._cfg["default"] = {
+            "id": component_id,
+            "component": self._COMPONENT_TYPE,
+            "version": str(self._CONFIG_VERSION),
         }
 
-        db_path  = os.path.join(self.root, VAR_FOLDER_NAME, f"{DB_PREFIX}{component_id}.json")
-        self._cfg['default']['db'] = os.path.relpath(db_path, os.path.join(self.root, CONFIG_FOLDER_NAME))
-
+        db_path = os.path.join(
+            self.root, VAR_FOLDER_NAME, f"{DB_PREFIX}{component_id}.json"
+        )
+        self._cfg["default"]["db"] = os.path.relpath(
+            db_path, os.path.join(self.root, CONFIG_FOLDER_NAME)
+        )
 
         ip, port = retrieve_ip_and_port(self.root)
-        allow_default_biprimes = os.getenv('ALLOW_DEFAULT_BIPRIMES', True)
+        allow_default_biprimes = os.getenv("ALLOW_DEFAULT_BIPRIMES", True)
```

Comment:
This line is not needed anymore

```suggestion
        allow_default_biprimes = os.getenv("ALLOW_DEFAULT_BIPRIMES", True)
```